05-cluster-prerequisites.md

```diff
@@ -35,7 +35,17 @@ Following the steps below will result in the provisioning of the shared Azure re
 
 1. Deploy the AKS cluster prerequisites and shared services.
 
-  > :book: The app team is provisioning an Azure Container Registry instance that is shared by their AKS clusters as well as their [Azure Monitor logs solution](https://docs.microsoft.com/azure/azure-monitor/logs/design-logs-deployment) and Azure Front Door ( Each client of our application around the world will be served for the closet AKS Cluster, and in case of some failure in one of the instance, the user will be served for another). The app team is creating multiple clusters for its new workload (Application Id: a0042). This array of clusters is a multi-region infrastructure solution composed by multiple Azure resources that regularly emit logs to Azure Monitor. All the collected data is stored in a centralized Log Analytics workspace for the ease of operations after confirming there is no need to split workspaces due to scale. The app team estimates that the ingestion rate is going to be less than `6GB/minute`, so they expect not to be throttled as this is supported by the default rate limit. If it was required, they could grow by changing this setup eventually. In other words, the design decision is to create a single Azure Log Analytics workspace instance in the `eastus2` region, and that is going to be shared among their multiple clusters. Additionally, there is no business requirement for a consolidated cross business units view at this moment, so the centralization is great option for them. Something the app team also considered while making a final decision is the fact that migrating from a _centralized_ solution to a _decentralized_ one can be much easier than doing it the other way around. As a result, the single workspace being created is a workload specific workspace, and these are some of the Azure services sending data to it:
+  > :book: The app team is about to provision three shared Azure resources. One is a non-regional and two regional, but more importantly they are deployed independently from their AKS Clusters:
+  >
+  > | Azure Resource                                                                                                | Non-Regional  |  East US 2  | Central US |
```

Comment:
I'm totes okay with the regions being called out explicitly like this. It matches the contents, and honestly we don't really give users that much control over the regions.  I mean, they could change params, but we haven't really "tested" that functionality or provided guidance on exactly what to change where.  So I'm all for what ya have here.

Reply:
thanks for this feedback. Please let me resolve this one based on this ^